src/config.rs

```diff
@@ -805,7 +805,12 @@ impl Config {
     /// 1. `$HOME/.config/alacritty.yml`
     /// 2. `$HOME/.alacritty.yml`
     pub fn load() -> Result<Config> {
-        let home = env::var("HOME")?;
+        let home = env::var("HOME")
+            .or_else(|err| if cfg!(windows) {
```

Comment:
I think it might make sense to hold off on Windows-specific logic and just add it all at once. This code path won't be tested by CI right now and nobody will be running it.